daemon/logs.go

```diff
@@ -79,11 +79,16 @@ func (daemon *Daemon) ContainerLogs(containerName string, config *backend.Contai
 			logrus.Errorf("Error streaming logs: %v", err)
 			return nil
 		case <-config.Stop:
-			logs.Close()
+			if !container.IsRunning() {
```

Comment:
are you sure this change is correct? I think we should close the log regardless the container is running or not when we get this signal.